app/src/main/java/org/fossasia/openevent/Adapters/TracksListAdapter.java

```diff
@@ -42,15 +53,31 @@ public int getItemCount() {
         return tracks.size();
     }
 
+    private void setAnimation(View viewToAnimate, int position) {
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
I was trying something, I'll remove this. For lastposition, if we reset lastPosition, then the animation would keep continuing, but my idea was that animation should be only for the first time the item was loaded.